seed/views/properties.py

```diff
@@ -642,12 +646,11 @@ def update(self, request, pk=None):
                 ).order_by('-id').first()
 
                 if 'extra_data' in new_property_state_data.keys():
-                    property_state_data['extra_data'].update(
-                        new_property_state_data.pop('extra_data'))
+                    property_state_data['extra_data'].update(new_property_state_data.pop('extra_data'))
                 property_state_data.update(new_property_state_data)
 
                 if log.name == 'Import Creation':
```

Comment:
These strings in the if blocks seem like voodoo, but it's not like you added them here, so OK.